content/200-concepts/100-components/01-prisma-schema/06-relations/07-referential-actions.mdx

```diff
@@ -0,0 +1,258 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions (Preview)'
+metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+In 2.26.0 and later, you can define referential actions on your relation fields. Referential actions determine what should happen to a record when a related record is deleted or updated.
+
+<Admonition type="warning">
+
+**If you have a project that is running version 2.25.0 and you then upgrade to 2.26.0 and enable the preview feature, using the `delete()` or `deleteAll()` methods may results in deletion of data where once it would not have.**
+
+</Admonition>
+
+When you enable the `referentialActions` preview flag, you must choose one of three upgrade paths. See the [Upgrading to referential actions guides](../../../../guides/upgrade-guides/upgrading-to-referential-actions) for more information on adopting the feature in your project
+
+</TopBlock>
+
+## What are referential actions?
+
+Referential actions are constraints that define how a referenced record is handled by the database when you run an [`update`](../../prisma-client/crud#update) or [`delete`](../../prisma-client/crud#delete) query.
```

Comment:
I tend to disagree here and find the definition high level yet accurate considering we're talking about Prisma concepts.
This could be indeed interesting to describe how that translates to the database level though

Reply:
To add more color, I think I may have not done a great job:

I think the use of the word constraint in this context could be confusing. We use foreign key constraints in the database and the constraints can be customized to trigger different actions on update and delete. 

When you introduce the high-level Prisma concept we should avoid using the word constraint as it has a meaning at the database level. Otherwise we are using it twice and this could cause confusing.

This is why I was suggesting to use a different word like policies.